var/spack/repos/builtin/packages/rpp/package.py

```diff
@@ -73,10 +107,10 @@ def patch(self):
         "opencv@4.5:"
         "+calib3d+features2d+highgui+imgcodecs+imgproc"
         "+video+videoio+flann+photo+objdetect",
-        type="build",
+        type=("build", "run"),
         when="@1.0:",
     )
-    depends_on("libjpeg-turbo", type="build")
+    depends_on("libjpeg-turbo", type=("build", "run"))
```

Comment:
Why does this package depend on `libjpegturbo` with `type=("build", "run")`, and not `type=("build", "link")`?

Reply:
changed to link